--- 3463 Silver Star
+++ 3463 Silver Star
@@ -1,7 +1,7 @@
 3463 Silver Star
 item      uncommon, Thrown, thrown
-level     ilvl 37
+level     ilvl 35
 binds     binds when picked up
-stats     +5 agility
-weapon    28 - 53 damage, speed 2.30, 17.6 dps
+stats     +4 agility
+weapon    26 - 49 damage, speed 2.30, 16.3 dps
 sells     9 copper
--- 6773 Kolkar Marauder Chain
+++ 6773 Kolkar Marauder Chain
@@ -1,6 +1,6 @@
 6773 Kolkar Marauder Chain
 item      uncommon, Mail, chest
-level     ilvl 42
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +14 strength, +7 agility
 sells     7366 copper
--- 6774 Traitor's Finger
+++ 6774 Traitor's Finger
@@ -1,6 +1,6 @@
 6774 Traitor's Finger
 item      uncommon, Miscellaneous, held in off hand
-level     ilvl 42
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +5 stamina, +5 intellect, +5 spirit
 sells     2885 copper
--- 6788 Kolkar Hunter's Belt
+++ 6788 Kolkar Hunter's Belt
@@ -1,6 +1,6 @@
 6788 Kolkar Hunter's Belt
 item      uncommon, Leather, waist
-level     ilvl 42
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +9 stamina, +8 agility
 sells     3331 copper
--- 6789 Ceremonial Centaur Blanket
+++ 6789 Ceremonial Centaur Blanket
@@ -1,6 +1,6 @@
 6789 Ceremonial Centaur Blanket
 item      uncommon, Cloth, back
-level     ilvl 42
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +2 intellect, +9 spirit
 sells     4012 copper
--- 271794 Centaur Spear
+++ 271794 Centaur Spear
@@ -1,7 +1,7 @@
 271794 Centaur Spear
 item      uncommon, Polearm, two-hand
-level     ilvl 43, requires level 1
+level     ilvl 38, requires level 35
 binds     binds when picked up
-stats     +11 strength, +13 stamina
-weapon    72 - 109 damage, speed 2.80, 32.3 dps
-sells     18116 copper
+stats     +9 strength, +11 stamina
+weapon    59 - 90 damage, speed 2.80, 26.6 dps
+sells     12329 copper
--- 271795 Wanderer's Broadsword
+++ 271795 Wanderer's Broadsword
@@ -1,7 +1,7 @@
 271795 Wanderer's Broadsword
 item      uncommon, Sword, two-hand
-level     ilvl 43, requires level 1
+level     ilvl 38, requires level 35
 binds     binds when picked up
-stats     +12 stamina, +10 hit rating
-weapon    62 - 94 damage, speed 2.40, 32.5 dps
-sells     18180 copper
+stats     +8 stamina, +10 hit rating
+weapon    51 - 77 damage, speed 2.40, 26.7 dps
+sells     12373 copper
--- 271796 Blade of the Magram Clan
+++ 271796 Blade of the Magram Clan
@@ -1,7 +1,7 @@
 271796 Blade of the Magram Clan
 item      uncommon, Sword, one-hand
-level     ilvl 43, requires level 1
+level     ilvl 38, requires level 35
 binds     binds when picked up
-stats     +4 stamina, +10 attack power
-weapon    38 - 72 damage, speed 2.20, 25 dps
-sells     13205 copper
+stats     +3 stamina, +9 attack power
+weapon    31 - 59 damage, speed 2.20, 20.5 dps
+sells     8987 copper
--- 271797 Staff of Revelations
+++ 271797 Staff of Revelations
@@ -1,7 +1,7 @@
 271797 Staff of Revelations
 item      uncommon, Staff, two-hand
-level     ilvl 43
+level     ilvl 38, requires level 35
 binds     binds when picked up
-stats     +10 intellect, +13 spirit
-weapon    78 - 117 damage, speed 3.00, 32.5 dps
-sells     16569 copper
+stats     +8 intellect, +11 spirit
+weapon    64 - 96 damage, speed 3.00, 26.7 dps
+sells     11276 copper
--- 271798 Desperate Barrier
+++ 271798 Desperate Barrier
@@ -1,6 +1,6 @@
 271798 Desperate Barrier
 item      uncommon, Shield, shield
-level     ilvl 43, requires level 1
+level     ilvl 38, requires level 35
 binds     binds when picked up
-stats     +5 strength, +8 stamina
-sells     8516 copper
+stats     +4 strength, +6 stamina
+sells     5796 copper
--- 271799 Desert Crawler's Claw
+++ 271799 Desert Crawler's Claw
@@ -1,7 +1,7 @@
 271799 Desert Crawler's Claw
 item      uncommon, Dagger, main hand
-level     ilvl 43
+level     ilvl 38, requires level 35
 binds     binds when picked up
-stats     +6 intellect, +2 stamina
-weapon    31 - 59 damage, speed 1.80, 25 dps
-sells     13358 copper
+stats     +5 intellect, +2 stamina
+weapon    25 - 48 damage, speed 1.80, 20.3 dps
+sells     9091 copper
--- 271800 Scavenged Magram Armament
+++ 271800 Scavenged Magram Armament
@@ -1,7 +1,7 @@
 271800 Scavenged Magram Armament
 item      rare, Polearm, two-hand
-level     ilvl 43, requires level 1
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +14 stamina, +12 attack power, +5 agility
-weapon    91 - 137 damage, speed 3.00, 38 dps
-sells     20114 copper
+weapon    89 - 134 damage, speed 3.00, 37.2 dps
+sells     18624 copper
--- 271801 Abandoned Ferocity
+++ 271801 Abandoned Ferocity
@@ -1,7 +1,7 @@
 271801 Abandoned Ferocity
 item      rare, Axe, two-hand
-level     ilvl 43, requires level 1
+level     ilvl 42, requires level 35
 binds     binds when picked up
-stats     +12 strength, +4 stamina, +10 agility
-weapon    82 - 123 damage, speed 2.70, 38 dps
-sells     20189 copper
+stats     +10 strength, +12 stamina, +4 agility
+weapon    95 - 143 damage, speed 3.20, 37.2 dps
+sells     18693 copper
--- 271802 Bludgeon of Betrayed Virtues
+++ 271802 Bludgeon of Betrayed Virtues
@@ -1,7 +1,7 @@
 271802 Bludgeon of Betrayed Virtues
 item      rare, Mace, one-hand
-level     ilvl 43, requires level 1
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +3 strength, +6 stamina, +2 agility
-weapon    45 - 84 damage, speed 2.20, 29.3 dps
-sells     16213 copper
+weapon    44 - 82 damage, speed 2.20, 28.6 dps
+sells     15012 copper
--- 271803 Greatstaff of the Necrokhans
+++ 271803 Greatstaff of the Necrokhans
@@ -1,7 +1,7 @@
 271803 Greatstaff of the Necrokhans
 item      rare, Staff, two-hand
-level     ilvl 43
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +12 intellect, +16 stamina, +6 spirit
-weapon    91 - 137 damage, speed 3.00, 38 dps
-sells     20343 copper
+weapon    89 - 134 damage, speed 3.00, 37.2 dps
+sells     18836 copper
--- 271804 Soulsplatter Mace
+++ 271804 Soulsplatter Mace
@@ -1,7 +1,7 @@
 271804 Soulsplatter Mace
 item      rare, Mace, main hand
-level     ilvl 43
+level     ilvl 42, requires level 35
 binds     binds when picked up
 stats     +6 intellect, +3 stamina, +1 spirit
-weapon    42 - 63 damage, speed 1.80, 29.2 dps
-sells     16336 copper
+weapon    41 - 62 damage, speed 1.80, 28.6 dps
+sells     15126 copper
--- 271805 Thrice-Damned Effigy
+++ 271805 Thrice-Damned Effigy
@@ -1,6 +1,6 @@
 271805 Thrice-Damned Effigy
 item      rare, Miscellaneous, held in off hand
-level     ilvl 43, requires level 1
+level     ilvl 42, requires level 35
 binds     binds when picked up
-stats     +8 spirit, +4 intellect, +2 stamina
-sells     6558 copper
+stats     +7 spirit, +4 intellect, +2 stamina
+sells     6072 copper
